ADApp/ntndArrayConverterSrc/ntndArrayConverter.cpp

```diff
@@ -90,8 +90,11 @@ NDColorMode_t NTNDArrayConverter::getColorMode (void)
     {
         if((*it)->getSubField<PVString>("name")->get() == "ColorMode")
```

Comment:
Maybe this should be `getSubFieldT` as well for consistency?

Reply:
I've done that now.  Please take a look at the new version.